src/main/java/pong/Paddle.java

```diff
@@ -1,4 +1,24 @@
 package pong;
 
 public class Paddle {
+    private int x, y;
+
+    public Paddle(Player player){
+        y = Environment.HEIGHT/2;
+        if (player.equals(Player.USER)){
+            //set human paddle to right side of board
+            x = Environment.WIDTH - PongView.PADDLE_WIDTH;
```

Comment:
Most of the example of pong I've seen have give the paddles a little cushion space so they're not right up against the wall of the frame which I kind of like. If you're for it, maybe give the paddles like 5 extra spaces? 
int PADDING = 5;
x = Environment.WIDTH - PongView.PADDLE_WIDTH - PADDING;
and 
x = PongView.PADDLE_WIDTH + PADDING;

Reply:
That is a good idea! Will do.

Reply:
I can do this in this Pr or another one that could name Paddle...

Reply:
you can include this with the paddle pr. Thx!